### DISCUSSION TITLE
Remove unused address to tabular server

### DISCUSSION THREAD

**Discussion by kamilsoszka** [DaxStudio/DaxStudio#1093] [Q&A] [answered]
Is it possible to remove the unused address for the tabular server ? 
The following image indicates the address I would like to get rid of:

![image](https://user-images.githubusercontent.com/70235786/230719153-24e42811-8863-4de2-9203-1259b12a75bc.png)

**Comment by dgosbell** [comment, marked as answer]
There is no easy way to do this at the moment. If you don't mind using RegEdit the list is stored at the following location `HKEY_CURRENT_USER\SOFTWARE\DaxStudio\ServerMRU` 

But otherwise it should not be too hard to add a delete button into the list box. I've created #1094 to track this request

**Reply by kamilsoszka** [reply]
Thank you very much for the advice, I took it. 

I am full of admiration when it comes to this tool.
